### PR TITLE
adding control and height for proper display of control panel fix #49

### LINKED ISSUE #49: Resizer doesn't consider the completion information
The h5p-resizing script doesn't take in consideration the completion activity information. 

- If we don't display it, we will see the control panel. 

- If we do display it, the control panel will be hidden at the bottom.

We need to add some control to adjust the height of the iframe to include the activity completion information.

### PR DESCRIPTION
**Pull request by gbarat87**
This is to replace this PR https://github.com/catalyst/moodle-mod_hvp/pull/50

### CODE REVIEW

File: amd/src/embed.js [hunk 1]
```diff
@@ -22,7 +22,10 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 var resizeDelay;
                 var instance = H5P.instances[0];
                 var parentIsFriendly = false;
-
+                var completionactivity = document.getElementsByClassName('activity-header')[0];
+                var comnpletionmargin = document.defaultView.getComputedStyle(completionactivity, '')
```
Comment (marcusboon) [inline amd/src/embed.js]: `comnpletionmargin` a typo?

File: amd/src/embed.js [hunk 2]
```diff
@@ -22,7 +22,10 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 var resizeDelay;
                 var instance = H5P.instances[0];
                 var parentIsFriendly = false;
-
+                var completionactivity = document.getElementsByClassName('activity-header')[0];
```
Comment (marcusboon) [inline amd/src/embed.js]: Are we calculating the `completionactivity` regardless of if completion is enabled or not? Would it be better to only calculate this if completion is enabled on the activity?

File: amd/src/embed.js (hunk 1 above)
Comment (gbarat87) [reply amd/src/embed.js]: I have corrected the typo

File: amd/src/embed.js (hunk 2 above)
Comment (gbarat87) [reply amd/src/embed.js]: I have added a control on the completion state here

**Review by marcusboon — COMMENTED** [review]
Can you please confirm that you've run grunt on the js to minify them? I think i've got a different version of composer or something so it's giving me changes when I run grunt

**Comment by gbarat87** [comment]
I did run the grunt. 
And I can see my update in the min.js and in the min.js.map

**Pull request merged**